README.md

```diff
@@ -111,7 +95,9 @@ Copyright (c) 2017 Calle Erlandsson, Anton Lindqvist & thoughtbot.
 [current]: https://github.com/calleerlandsson/pick/blob/master/CONTRIBUTING.md
 [debian]: https://packages.debian.org/stretch/pick
```

Comment:
This should point to: https://packages.debian.org/stable/misc/pick

Reply:
Fixed in 65db1c9.